arch/arm64/boot/dts/rockchip/Makefile

```diff
@@ -389,6 +389,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-radxa-cm5-io.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-radxa-cm5-rpi-cm4-io.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-radxa-e52c.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-radxa-nx5-io.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-retro-lite-cm5.dts
```

Comment:
Should be .dtb. devicetree is not build in workflow.

Reply:
Oh crikey, thank you for spotting.

Reply:
Fixed.